Eshop.WebApi/Controllers/ProductController.cs

```diff
@@ -7,10 +7,16 @@ namespace Eshop.WebApi.Controllers
     [Route("[controller]")]
     public class ProductController : Controller
     {
+        private static List<Category> Categories = new List<Category>
+        {
+            new Category(1, "Computers"),
+            new Category(2, "Mouses")
+        };
+
         private static List<Product> Products = new List<Product>
         {
-            new Product(1, "Notebook Acer 16", "Best notebook out there", 399.99m),
-            new Product(2, "Mouse Razor 123", "Best Mouse", 14.50m),
+            new Product(1, "Notebook Acer 16", "Best notebook out there", 399.99m, Categories[1]),
```

Comment:
Category[0], you see we have no tests and you did not test it properly so it is a bug.

Reply:
You have not commited it. It is probably only on your local. You can see it also in commit - you can not win this.
Check it properly.